disc/cog.py

```diff
@@ -14,6 +19,58 @@ class Descriptions:
 class Slash(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.api = Linkedin(LINKEDIN_ACCOUNT, LINKEDIN_PASSWORD)
+
+    @cog_ext.cog_slash(name="view-profile", description=Descriptions.POST_JOB)
+    async def _view_profile(self, ctx: SlashContext, profile):
+
+        await ctx.send("Hang on, grabbing details for you...", hidden=True)
+
+        get_profile = self.api.get_profile(profile)
+
+        time_string = ""
+        for job in get_profile['experience']:
+            time_period = job['timePeriod']
+            if 'endDate' in time_period:
+                time_string += (
+                    f"**🚀 {job['companyName']}** ({time_period['startDate']['month']},"
+                    f"{time_period['startDate']['year']} to {time_period['endDate']['month']}, "
+                    f"{time_period['endDate']['year']})\n*{job['title']}*\n\n"
+                    
+                )
+            else:
+                time_string += (
+                    f"**🚀 {job['companyName']}** ({time_period['startDate']['month']}, "
+                    f"{time_period['startDate']['year']} to Present)\n*{job['title']}*\n\n"
+                )
+        
+        school_string = ""
+        for education in get_profile['education']:
+                school_string += f"**🎖 {education['schoolName']}** - {education['fieldOfStudy']}\n\n"
+                
+
+        description = (
+            f"**{get_profile['headline']}**\n\n"
+            f"Based in {get_profile['geoLocationName']}, {get_profile['location']['basicLocation']['countryCode'].upper()}, "
+            f"{get_profile['firstName']} is currently working in the **{get_profile['industryName']}** industry.\n\n"
+            f"__Here's where {get_profile['firstName']} worked:__\n\n"
+            f"{time_string}__Here's where {get_profile['firstName']} studied:__\n\n{school_string}"
+        )
+        
+        colors = Colors()
+        random_color = colors.random_color()
+        final_color = discord.Colour(random_color)
+
+        embed_content = discord.Embed(
+            title=f"A look at {get_profile['firstName']} {get_profile['lastName']}'s profile.",
+            type="rich",
+            description=description,
+            color=final_color
+        )
+
+        embed_content.set_thumbnail(url=get_profile['displayPictureUrl'] + get_profile['img_100_100'])
```

Comment:
Did we make sure 100x100 is always an option? Should we consider dynamically grabbing the image size instead?